lib/internal/test_runner/runner.js

```diff
@@ -435,8 +435,11 @@ function run(options) {
   }
   if (shard != null) {
     validateObject(shard, 'options.shard');
-    validateNumber(shard.total, 'options.shard.total', 1);
-    validateNumber(shard.index, 'options.shard.index');
+    // Avoid re-evaluating the shard object in case it's a getter
+    shard = { index: shard.index, total: shard.total };
```

Comment:
```suggestion
    shard = { __proto__: null, index: shard.index, total: shard.total };
```

Reply:
is there a reason there is no eslint rule for that?

Reply:
I don't think anybody's taken the time to make one. There will be a few places where objects *shouldn't* have a null prototype - mostly when they're returned to userland - but those could have override comments.

Reply:
I'll make one

Reply:
https://github.com/nodejs/node/pull/48646 need to answer some questions before changing all files